### PR TITLE
[CI] Make tests green again

### PR DESCRIPTION
**Pull request by vtsykun**
| Q             | A
| ------------- | ---
| Branch?       | 6.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | -
| License       | MIT
| Doc PR        | -

### CODE REVIEW

File: src/Symfony/Component/Cache/Traits/RelayProxy.php [hunk 1]
```diff
@@ -456,6 +456,11 @@ public function bitcount($key, $start = 0, $end = -1, $by_bit = false): \Relay\R
         return ($this->lazyObjectState->realInstance ??= ($this->lazyObjectState->initializer)())->bitcount(...\func_get_args());
     }
 
+    public function bitfield($key, ...$args): \Relay\Relay|array|false
```
Comment (Nyholm) [inline src/Symfony/Component/Cache/Traits/RelayProxy.php:459]: hm.. why was this method added?

**Review by Nyholm — CHANGES_REQUESTED** [review]
Thank you for this PR. It seams to include more than "make CI green". Or am I wrong?

File: src/Symfony/Component/Cache/Traits/RelayProxy.php (hunk 1 above)
Comment (vtsykun) [reply src/Symfony/Component/Cache/Traits/RelayProxy.php:459]: The test `Symfony\Component\Cache\Tests\Traits\RedisProxiesTest::testRelayProxy` [failed](https://github.com/symfony/symfony/actions/runs/6138815527/job/16656061403#step:8:8501) after update php extension `ext-relay`, just I sync proxy trait with latest changes of `Relay\Relay` class

File: src/Symfony/Bundle/SecurityBundle/composer.json [hunk 2]
```diff
@@ -29,7 +29,7 @@
         "symfony/password-hasher": "^5.4|^6.0|^7.0",
         "symfony/security-core": "^6.2|^7.0",
         "symfony/security-csrf": "^5.4|^6.0|^7.0",
-        "symfony/security-http": "^6.3|^7.0"
+        "symfony/security-http": "^6.3.4|^7.0"
```
Comment (vtsykun) [inline src/Symfony/Bundle/SecurityBundle/composer.json]: Related to issue https://github.com/symfony/symfony/pull/51104

result before. https://github.com/symfony/symfony/actions/runs/6138815527/job/16656061481#step:8:1778

File: src/Symfony/Component/Messenger/Bridge/Doctrine/Tests/Transport/DoctrineIntegrationTest.php [hunk 3]
```diff
@@ -56,6 +57,10 @@ public function testConnectionSendAndGet()
 
     public function testSendWithDelay()
     {
+        if (interface_exists(Result::class)) {
+            $this->markTestSkipped('Skip on Doctrine DBAL 2');
+        }
```
Comment (derrabus) [inline src/Symfony/Component/Messenger/Bridge/Doctrine/Tests/Transport/DoctrineIntegrationTest.php]: This does not feel right and looks like something I might have broken. Let me check that.

Comment (derrabus) [reply src/Symfony/Component/Messenger/Bridge/Doctrine/Tests/Transport/DoctrineIntegrationTest.php]: #51624

File: src/Symfony/Bundle/SecurityBundle/composer.json (hunk 2 above)
Comment (derrabus) [reply src/Symfony/Bundle/SecurityBundle/composer.json]: This change should go to the 6.3 branch then.

File: src/Symfony/Component/Messenger/Bridge/Doctrine/Tests/Transport/DoctrineIntegrationTest.php (hunk 3 above)
Comment (derrabus) [reply src/Symfony/Component/Messenger/Bridge/Doctrine/Tests/Transport/DoctrineIntegrationTest.php]: Please revert this change. The test should pass now with DBAL 2 as well.

File: src/Symfony/Bundle/SecurityBundle/composer.json (hunk 2 above)
Comment (vtsykun) [reply src/Symfony/Bundle/SecurityBundle/composer.json]: Done, moved to PR https://github.com/symfony/symfony/pull/51627

Comment (nicolas-grekas) [reply src/Symfony/Bundle/SecurityBundle/composer.json]: Merged. Can you please rebase this one?

Comment (vtsykun) [reply src/Symfony/Bundle/SecurityBundle/composer.json]: Done

**Comment by vtsykun** [comment]
Looks like the random bug #51105 is back again. 
https://github.com/symfony/symfony/actions/runs/6149268135/job/16684768491#step:8:8976

**Review by nicolas-grekas — APPROVED** [review]

**Comment by nicolas-grekas** [comment]
Thank you @vtsykun.

**Comment by nicolas-grekas** [comment]
(I cherry-picked the changes that applied to 5.4 and 6.3 respectively)

**Pull request merged**